### ISSUE TITLE
Add Fedora 26 (and 27?)

### ISSUE THREAD

**Issue by rvagg** [nodejs/build#963]
https://fedoraproject.org/wiki/Releases

27 has slipped a bit but is due on the 7th if I'm reading this right: https://fedoraproject.org/wiki/Releases/27/Schedule

**Comment by refack** [comment]
> 27 has slipped a bit but is due on the 7th if I'm reading this right:

That's what I'm reading, but https://bugzilla.redhat.com/showdependencytree.cgi?id=F27FinalBlocker&hide_resolved=1 shows quite a few blockers still open. So 🤞

**Comment by rvagg** [comment]
So I've added 26, one on DO and one on Rackspace. Landed Ansible updates @ 326ca9d (pretty trivial, added to inventory and extended 3 fedora24 & fedora25 special cases in ansible/roles/bootstrap/tasks/partials/fedora.yml).

I've also added a couple of new labels in Jenkins: `fedora-latest-x64` on `fedora26-x64` and `fedora-last-latest-x64` on `fedora25-x64`. I've then gone through all of the job config files (on the server, not via the web interface) and done some tweaking: where fedora25 exists, I've added fedora26 as well (mostly the node-test-commit-linux jobs and its copies/variants). Where only fedora22 and fedora23 exists (a lot of jobs) I've switched them to fedora-last-latest-x64 and fedora-latest-x64. This is the case for most of the citgm jobs, they've been running on very out of date Fedora and I've gone ahead and made the assumption that we'd rather them run on actively supported Fedora and not have to manually upgrade the labels in all of those jobs.

Fedora 27 isn't available yet but when I remember I'll add it.

/cc @nodejs/jenkins-admins @MylesBorins
